modules/aws_asg/high-cpu-check.bash

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+__logger() {
+  level="${1}" && shift
+  msg="${1}" && shift
+  date="$(date --iso-8601=seconds)"
+  log_msg="tag=cron time=${date} level=${level} msg=\"${msg}\""
```

Comment:
Ack, sorry again!  Please declare `level`, `msg`, `date`, and `log_msg` as `local`, too :heart: